ml-agents/mlagents/trainers/tests/test_learn.py

```diff
@@ -156,6 +155,12 @@ def test_commandline_args(mock_file):
     assert opt.engine_settings.no_graphics is True
     assert opt.debug is True
     assert opt.checkpoint_settings.inference is True
+    assert opt.checkpoint_settings.resume is False
+
+    # ignore init if resume set
+    full_args.append("--resume")
+    opt = parse_command_line(full_args)
+    assert opt.checkpoint_settings.initialize_from is None  # ignore init if resume set
     assert opt.checkpoint_settings.resume is True
 
```

Comment:
Do we need a test to check if they were both set in the YAML but not the CLI? I think we can do this in the `test_settings.py`.